app/assets/styles/_application-variables.scss

```diff
@@ -67,3 +67,9 @@ $button-horizontal-padding:    15px;
 $app-ResourceCard-height: 180px;
 $app-SearchPage-max-width: 800px;
 $app-SearchPage-margin: 10px;
+
+/* BREAKPOINTS */
+$screen-sm-min: 576px;
```

Comment:
We use bootstrap grid system. Can you explain the reason behind extra those breakpoints?
In the future, varaamo will use something call Helsinki Design System, that including universal layout for all Helsinki application.

TLDR: this is quite unneccesary